### PR TITLE
Numerous fixes to support 3.7.

### PR DESCRIPTION
**Pull request by kacfengine**
* Update supported platforms
* Update main page.
* Update upgrade procedure.
* Update known issues.
* Delete What's New and simply point it to the changelog.
* Fix some typos and some stylistic things.

### CODE REVIEW

**Comment by kacfengine** [comment]
@estenberg, @nickanderson: Perhaps you could take a look at this?

This is kind of a bare minimum, things can certainly be improved, but at least we bring it to a tolerable level that doesn't make it look like we just released 3.6.

File: guide/installation-and-configuration/upgrading-to-37.markdown [hunk 1]
```diff
@@ -73,38 +156,31 @@ If your clients get promise failures (not kept) similar to "Can't stat file '/va
...
-  * Download the [`cfmigrate`](http://cfengine.package-repos.s3.amazonaws.com/tools/cfmigrate) binary.
-  * This binary will export user/role settings as well as long-living file-changes data from MongoDB.
-  * No other data will be exported, as it would either way expire after
-    one week. If you need continued access to 3.5 compliance data,
-    keep the 3.5 MongoDB available.
-  * The `cfmigrate` tool uses `mongoexport`, which is available from [mongodb.org](http://www.mongodb.org/downloads) (it is included in the hub package of CFEngine 3.5).
-4. On your existing Policy Server, stop the CFEngine services.
+    in the merged masterfiles policy, e.g. /root/3.7/masterfiles/update/update_policy.cf`.
+
+    Since 3.6, CFEngine uses LMDB for local databases, whereas older versions of CFEngine typically use TokyoCabinet or QDBM. The classic networking protocol uses the `lastseen` database to verify that the mapping between a peer's IP address and the corresponding hostkey is not changed. Since the 3.7 installation will not have any mappings in the lastseen database, hosts won't trust the IP address of the policy server without that setting.
```
Comment (nickanderson) [inline guide/installation-and-configuration/upgrading-to-37.markdown]: I think this paragraph can be completely removed cant it? 3.6 had LMDB, so 3.7 wont have this issue of missing entries.

File: guide/installation-and-configuration/upgrading-to-37.markdown [hunk 2]
```diff
@@ -120,21 +196,21 @@ If everything looks good, you are ready to upgrade the clients, please skip to P
...
 3. Install the new CFEngine Policy Server package (you may need to adjust the package name based on CFEngine edition, version and distribution).
   * ```console
-    rpm -i cfengine-nova-hub-3.6.1.x86_64.rpm # Red Hat based distribution
+    rpm -i cfengine-nova-hub-3.7.0.x86_64.rpm # Red Hat based distribution
     ```
   * ```console
-    dpkg --install cfengine-nova-hub_3.6.1-1_amd64.deb # Debian based distribution
+    dpkg --install cfengine-nova-hub_3.7.0-1_amd64.deb # Debian based distribution
     ``` 
 4. Restore ppkeys and any Enterprise license file (default location is /var/cfengine/masterfiles/license.dat) from backup.
```
Comment (nickanderson) [inline guide/installation-and-configuration/upgrading-to-37.markdown]: I dont think we have to restore the keys, we arent having them uninstall the hub package anymore right?

File: guide/latest-release/known-issues.markdown [hunk 3]
```diff
@@ -106,8 +97,8 @@ Monitoring graphs are not supported on all platforms, currently Aix and Windows
 
 
 ### Enterprise reports not collected from 3.5
-CFEngine Enterprise 3.6 has a new diff-based report collection mechanism,
-and so a 3.6 hub cannot collect reports from 3.5 or earlier agents.
+CFEngine Enterprise 3.7 has a new diff-based report collection mechanism,
```
Comment (nickanderson) [inline guide/latest-release/known-issues.markdown]: 3.6 has the new diff-based report collection mechanism.

File: guide/latest-release/whatsnew/changelog-enterprise.markdown [hunk 4]
```diff
@@ -3,7 +3,7 @@ layout: default
 title: Enterprise ChangeLog
 published: true
 sorting: 20
-tags: [latest release, "3.6", what's new, enterprise]
+tags: [latest release, "3.7", what's new, enterprise]
 ---
```
Comment (nickanderson) [inline guide/latest-release/whatsnew/changelog-enterprise.markdown]: We need to fill out this enterprise changelog still

Comment (nickanderson) [inline guide/latest-release/whatsnew/changelog-enterprise.markdown]: Never mind, i see it includes the enterprise changelog.

File: guide/installation-and-configuration/upgrading-to-37.markdown (hunk 1 above)
Comment (kacfengine) [inline guide/installation-and-configuration/upgrading-to-37.markdown]: Indeed, you're right.

File: guide/installation-and-configuration/upgrading-to-37.markdown (hunk 2 above)
Comment (kacfengine) [inline guide/installation-and-configuration/upgrading-to-37.markdown]: That section is still there, but yes, it should be unneeded now, I removed both of them, and also the mention of importing from MongoDB just below this section.

**Comment by kacfengine** [comment]
Alright, I have addressed all the issues and updated. I also removed tha part that says you should switch to TLS, since TLS is the default in 3.7.

If there are no more comments by Monday, I'll merge it then.

**Pull request merged**